### PR TITLE
Supporting Rails 4.2 update on continuous cover

### PR DESCRIPTION
**Pull request by joseairosa**
### Description

Part of https://github.com/simplybusiness/continuous_cover/pull/920.

### Sign-off

- [ ] Tech - @amar-shah @joshuafleck

### CODE REVIEW

File: devise_ldap_authenticatable.gemspec [hunk 1]
```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency('rake', '>= 0.9')
   s.add_development_dependency('rdoc', '>= 3')
-  s.add_development_dependency('rails', '>= 4.0')
```
Comment (joshuafleck) [inline devise_ldap_authenticatable.gemspec]: Why is this change necessary?

**Comment by amar-shah** [comment]
Also points to question should we be merging forked version of branch into our version

https://github.com/cschiewek/devise_ldap_authenticatable/blob/master/devise_ldap_authenticatable.gemspec#L26

**Comment by joseairosa** [comment]
> Also points to question should we be merging forked version of branch into our version

Possibly. Altho I don't know exactly what lead us to fork in the first place, but I've a feeling that it had to do with some special requirements from our LDAP config?

File: devise_ldap_authenticatable.gemspec (hunk 1 above)
Comment (joseairosa) [inline devise_ldap_authenticatable.gemspec]: I think I changed this in the attempt to test something and ended up forgetting to rollback

**Comment by amar-shah** [comment]
@joseairosa yep I think current gem does not support going to backup of LDAP server.
